packages/replay-next/components/console/filters/FilterToggles.tsx

```diff
@@ -82,21 +82,28 @@ export default function FilterToggles() {
       )}
       <div className={styles.EmptySpaceFiller} />
       <hr className={styles.Divider} />
-      <Toggle
-        afterContent={
-          <Suspense fallback={null}>
-            <NodeModulesCount />
-          </Suspense>
-        }
-        checked={showNodeModules}
-        label="Node modules"
-        onChange={showNodeModules => update({ showNodeModules })}
-      />
-      <Toggle
-        checked={showTimestamps}
-        label="Timestamps"
-        onChange={showTimestamps => update({ showTimestamps })}
-      />
+
+      <div className="py-1">
```

Comment:
This change caused a small regression in the console.
<img width="316" alt="Screen Shot 2023-03-14 at 5 15 05 PM" src="https://user-images.githubusercontent.com/29597/225137951-a6983834-1c3f-4dbd-8d8f-fb6637316227.png">

Can we avoid using Tailwind styles inside of the "replay-next" package? At least for consistently within this package– we're using CSS modules.